### ISSUE TITLE
Feature request: 

### ISSUE THREAD

**Issue by JRMeyer** [coqui-ai/STT#1879]
**Is your feature request related to a problem? Please describe.**
Yes. I find myself saving training logs to a text file (what I consider) a hacky style. On a server, using the docker training image, I find myself always doing this: 

```
$ docker run train.py &> /my/local/file.log &
```

This seems non-obvious to me, and I see people training models in docker containers, and then they're surprised when the training logs and loss data is gone. Saving the Loss curves to a text file makes viewing the training progress much easier on a remote server than using tensorboard, for example.

**Describe the solution you'd like**
I'd like to be able to save training loss information to a text file with a command-line flag such as `--training_log_file /path/to/file.txt`

**Describe alternatives you've considered**
Redirecting output from stdout to a text file (as mentioned above). Inspecting tensorboard graphs (but I haven't tried this on a remote server, and afaik you can't access them during training.) I like to keep an eye on training curves during training with `gnuplot`, by piping a single col of (train or dev) losses into `| gnuplot -p -e "set terminal dumb $(tput cols) $(tput lines); plot '-' using 1"`

**Additional context**
None
